temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -130,4 +130,16 @@ message Namespace {
     // The date and time when the namespace was last modified.
     // Will not be set if the namespace has never been modified.
     google.protobuf.Timestamp last_modified_time = 11;
+    // The status of each region where the namespace is available.
+    // The id of the region is the key and the status is the value of the map.
+    map<string, NamespaceRegionStatus> region_status = 12;
+}
+
+message NamespaceRegionStatus {
+    // The current state of the namespace region.
```

Comment:
[Nit] might want to reword this a bit as this could be interpreted as "the state of the whole region the namespace is in"
Maybe, the current state of the namespace in the given region or something would be better?

Reply:
This is the state of a region where the namespace is in.